package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "scripts": {
+    "format": "prettier --check .",
+    "format:fix": "prettier --write ."
+  },
+  "devDependencies": {
```

Comment:
I'm not sure weather I should also add eslint-related dependencies and scripts in this package.json file.